Brooks/Example/FileTests.hs

```diff
@@ -0,0 +1,38 @@
+module Example.FileTests (fileTests) where
```

Comment:
I think this would fit better under `code/CodeGen/Test/FileTests`. And the same with most of `Brooks/Example` (I'll assume so, and comment explicitly on exceptions).